examples/agentic_rl/react_agent/README.md

```diff
@@ -0,0 +1,32 @@
+# Train ReAct Agent with Reinforcement Learning
+
+This example demonstrates how to train the ReAct agent using reinforcement learning (RL). Here we use [Trinity-RFT](https://github.com/modelscope/Trinity-RFT) as the training framework and train the agent on the GSM8K dataset.
```

Comment:
==> [Trinity-RFT] is a xxx framework. In this example, we demonstrate how to train a agent built by AgentScope in Trinity-RFT.